src/ops/direnv/envrc.bash

```diff
@@ -120,6 +120,9 @@ function declare() {
         # vars from: https://github.com/NixOS/nix/blob/92d08c02c84be34ec0df56ed718526c382845d1a/src/nix-build/nix-build.cc#L421
         "NIX_ENFORCE_PURITY="*) punt;;
 
+        # https://github.com/target/lorri/issues/97
+        "preHook="*) punt;;
```

Comment:
Another possible fix we could take is capturing the original preHook in the env vars, and assign `preHook=origPreHook` in the envrc.bashrc. What do you think?

Reply:
If we are overwriting the original `preHook` atm, we should definitely restore it afterwards.